intelmq/bots/experts/cut_from_string/expert.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+Cut from string
+"""
+from intelmq.lib.bot import Bot
+
+
+class CutFromStringExpertBot(Bot):
+    string_from_start: int = 1  # 1 - from start, 0 - from end
+    string_for_cut: str = 'www.'
+    field_for_cut: str = 'source.fqdn'
+
+    def init(self):
+        pass
+
```

Comment:
If this method doesn't do anything, we maybe simple can remove it?

Reply:
I will remove it.